packages/cli/src/controller/generate-controller.spec.ts

```diff
@@ -287,7 +287,7 @@ describe('CLI codegen:generate', () => {
   });
   it('filter out existing methods, input address === undefined && ds address === "", should filter', () => {
     const ds = mockDsFn();
-    ds[0].options.address = '';
+    ds[0].options = Object.assign(ds[0].options ?? {}, {address: ''});
```

Comment:
Again, i think in tests its easier to use non-null assertions
```suggestion
    ds[0].options!.address = '';
```

Reply:
If changed this way, ESlint checks will report an error.